### ISSUE TITLE
hello sir are you here i need your help i am facing some problem while using your do-dmart scraper

### ISSUE THREAD

**Issue by arpitdayma** [octate/go-dmart#1]

**Comment by arpitdayma** [comment]
<img width="960" alt="Screenshot 2022-03-03 133050" src="https://user-images.githubusercontent.com/80608220/156522161-75fb1a0e-b517-4116-8286-3a87cc73b213.png">
  

facing these problem

**Comment by octate** [comment]
Hey @arpitdayma , make sure you have cloned the repo and then run the program. The missing structs are defined in models.go file

**Comment by arpitdayma** [comment]
sir thanks for your reply but i am new in these filed i don't have any knowlage about it. can you plz tell me step by step what to do? in first step i have to cloned the repo? and what is repo?

**Comment by arpitdayma** [comment]
yes sir the missing structs are defined in models.go file but how i can connect it with main.go ?

**Comment by arpitdayma** [comment]
[](url
<img width="960" alt="Screenshot 2022-03-04 110517" src="https://user-images.githubusercontent.com/80608220/156705953-d18f02bb-8670-4306-a092-8c43aa00a478.png">
)

sir i have cloned the repo also

**Comment by arpitdayma** [comment]
<img width="960" alt="Screenshot 2022-03-04 130046" src="https://user-images.githubusercontent.com/80608220/156719237-fadd21a4-0ecf-4a80-a66e-987dc8113ea9.png">


sir i have to edit any think here? sir plz reply it will really help me alot

**Comment by octate** [comment]
run the program using `go run .` and not `go run main.go`
if it does not work then:
try to tidy the mod file.
`go mod tidy`
`go mod download`
and then run the program

**Comment by arpitdayma** [comment]
<img width="960" alt="Screenshot 2022-03-04 162404" src="https://user-images.githubusercontent.com/80608220/156750634-42aa030d-7ebc-4461-8815-7bc8b743bf60.png">


sir it really work osm but how i can change the pin?? means i want to change the pincode

**Comment by arpitdayma** [comment]
sir plz reply i want tp scrape data of these pincode - 452001 not of 400053

**Comment by octate** [comment]
Hi @arpitdayma 
I missed the notification. Were you able to get through your requirement?
